docs/advanced-topics/reconnecting-mid-game.md

```diff
@@ -0,0 +1,51 @@
+---
+id: reconnecting-mid-game
+title: Reconnecting Mid-Game
+---
+
+In a multiplayer game, clients may get disconnected from the server for a variety of reasons (network issues, application/device crashes, etc). For those reasons, you may want to allow your players to reconnect to the game.
+
+## Considerations
+
+Review the following considerations before allowing clients to reconnect in the middle of an active game:
+
+- Clients may have specific state and in-game data associated with them. If you want them to reconnect with that same state and data, implement a mechanism to maintain the association of state and data with the client for when they reconnect. See [Session Management](session-management.md) for more information.
+- When using [scene management](../basics/scene-management.md), remember that the synchronization process launches when connecting to a game. Therefore, all additive scenes currently loaded on the server will load without unloading anything on the client when the client's active main scene is the same as the server's. <!-- What is the negative affect of this? Like is it slower loading/initial lag, etc? Or is it just cleaner and better practice? -->
+  - If you want to avoid this behavior, you can
+    - Unload those scenes when a client disconnects
+    - Unload those scenes when a client reconnects
+    - Use the `NetworkSceneManager.VerifySceneBeforeLoading` callback to prevent loading scenes already loaded on the client
+- Depending on your game's requirements, make sure the client's state is properly reset and ready to connect before attempting reconnection. This may require resetting some external services.
+
+
+## Automatic reconnection
+
+For a smoother experience for your players, clients can automatically attempt to reconnect to a game when connection is lost unexpectedly.
+
+To implement automatic reconnection: <!-- Does order matter for these steps -->
```

Comment:
They're not really steps, more like guidelines to follow, so the order does not matter